PODS with MARS 160g (Australia, 06.10.17)/pods_with_mars.txt

```diff
@@ -0,0 +1,19 @@
+Product_Name; PODS with MARS (160g)
```

Comment:
you should change the (160g) -> (100g) to match the data below. Would be more explicit.

Reply:
@Justicimo 160g is the correct input. We are talking about the product's actual weight